examples/EnableServiceProfilerInVSCLR2_1/README.md

```diff
@@ -22,30 +25,29 @@ At the end of the wizard, it is expected to see Application Insights being fully
 ![Application Insights Fully Configured](./.media/ApplicationInsightsConfigured.png)
 
 ## Add a reference to ServiceProfiler
+
 Right-click on the project in Solution Explorer, choose `Manage NuGet Packages...` and the reference to the NuGet Package of `Microsoft.ApplicationInsights.Profiler.AspNetCore`.
 
 ![Add NuGet package of Service Profiler](./.media/AddNuGetPackageOfServiceProfiler.png)
 
-Then, we need to add the environment variable of `ASPNETCORE_HOSTINGSTARTUPASSEMBLIES` to the `docker-compose.yaml` like this:
-```yaml
-version: '3.4'
-
-services:
-  appinsightsprofilerexample:
-    image: ${DOCKER_REGISTRY}appinsightsprofilerexample
-    environment:
-      "ASPNETCORE_HOSTINGSTARTUPASSEMBLIES": "Microsoft.ApplicationInsights.Profiler.AspNetCore"
-    build:
-      context: .
-      dockerfile: AppInsightsProfilerExample/Dockerfile
+Then, turn on Service Profiler in `Startup.cs`:
+
+```csharp
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // This statement adds Profiler to the application.
+            services.AddServiceProfiler();
+            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+        }
 ```
+
 Save and build the project. F5 for a Debug. In the output window, it is expected to see the Service Profiler logs in the Debug level:
 
 ![Service Profiler logs in VS Output Window](./.media/ServiceProfilerLogInVSOutput.png)
 
 Keep the application running for 2 minutes, access the website to generate some traffic. At the end of the session, you will see the logs like it below:
 
-```log
+```bash
```

Comment:
Does this do anything? I do not see "log" nor "bash" on [the list of GitHub-recognized grammars](https://github.com/github/linguist/blob/master/vendor/README.md)

Reply:
Take it back, `bash` is an alias for `shell`. Still, this is not bash ☺️

Reply:
Shell is better.